pkg/apis/core/v1alpha1/extensions_schedulingprofile.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/kubewharf/kubeadmiral/pkg/controllers/scheduler/framework/plugins/names"
 )
 
-func GetDefaultEnabledPlugins() *fedcore.EnabledPlugins {
+func GetDefaultEnabledPlugins(replicasPlugin string) *fedcore.EnabledPlugins {
```

Comment:
Any chance to mutate the returned `EnabledPlugins` in the scheduler? The default list is the baseline, and should not vary depending on configuration.